tests/storage/cdi_clone/test_clone.py

```diff
@@ -114,8 +114,8 @@ def test_successful_clone_of_large_image(
         pytest.param(
             {
                 "dv_name": "dv-source",
-                "image": f"{Images.Cirros.DIR}/{Images.Cirros.QCOW2_IMG}",
-                "dv_size": Images.Cirros.DEFAULT_DV_SIZE,
+                "image": f"{Images.Fedora.DIR}/{Images.Fedora.FEDORA41_IMG}",
```

Comment:
It would be even better to use `FEDORA_LATEST["image_path"]` like you did for RHEL images, so we wouldn't need to update the tests when a new OS version comes up.

Please use it here and in all other places in this PR